shared/oae/api/oae.api.util.js

```diff
@@ -701,6 +701,11 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
             if (/https?:\/\//i.test(displayName)) {
```

Comment:
I noticed that the form validation in the `userdetails` widget does not appear to be using `isValidDisplayName` for validation of the entered display name. Because of that, it's now possible to add names containing `http(s)` or `@`.